client-lib/src/main/java/com/ibm/ws/repository/resources/writeable/RepositoryResourceWritable.java

```diff
@@ -303,6 +303,15 @@
      */
     public void setLicenseId(String lic);
 
+    /**
+     * Set the maven coordinates
+     * <p>
+     * This is only set for resources which are stored in a maven repository.
+     * <p>
+     * Maven coordinates are returned as a string in the format {@code groupId:assetId:version}.
```

Comment:
Iffy javadoc. This is a setter not a getter

Reply:
I think it's ok. It's a getter for a field, but the person calling the getter needs to know that they shouldn't expect it to be set unless the resource is stored in a maven repository.